ISISScientificComputing/autoreduce
# New Webapp interface

### Summary of work
* Added Debug toolbar - this should only be rendered in debug mode 
* Added landing page `/overview`
* Set `/overview` to be the first page after redirect or accessing index
* Created new instrument pages that show variables, run details and status
* Added custom Pagination for run navigation

### How to test your work
#### Test Index
* Boot the web app and ensure the landing page is `/overview`

#### Test Instrument pages
* Go to an instrument page
* Try everything that you can break things:
  * Ensure links work
  * Ensure pagination works
  * Ensure filtering works
  * Ensure drop down menus work for instrument variables
  * Ensure drop down menus work for experiments

Fixes #298
